lte/protos/session_manager.proto

```diff
@@ -540,3 +556,226 @@ service CentralSessionController {
+   string dnn_value = 1;
+   repeated TrackingAreaIdentityList identity_list = 2;
+}
+message TrackingAreaIdentityList {
+   repeated TypeOfList type_of_list = 1;
+   repeated int32 number_of_elements = 2;
+   string MCC=3;
+   string MNC=4;
+   string TAC= 5;          // tac is havaing length 8 bytes,MCC is 3 bytes, MNC is 3 bytes
+}
+enum TypeOfList  {
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_NON_CONSECUTIVE_TAC_VALUES = 0;
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_CONSECUTIVE_TAC_VALUES = 1;
+   LIST_OF_TAIS_BELONGING_TO_DIFFERENT_PLMNS = 2;
+}
+enum priorityaccess  {
+   High = 0;
+   Normal = 1;
+   Low = 2;
+}
+
+enum SMSessionFSMState {
+        CREATING_0 = 0;
+        CREATE_1 = 1;
+        ACTIVE_2 = 2;
+        INACTIVE_3 =3;
+        RELEASED_4 = 4;
+}
+
+
+//PDU session Establishment from AMF to SMF
+//SET MESSSAGE
+message M5GSMSessionContext {
+  bytes pdu_session_id = 1;   //One byte
+  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
+  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  priorityaccess priority_access = 4;
+  AccessType access_type = 5;     //Define enum TS-29.571 - section 5.4.3
+  SubscriberID sid = 25;
+  string imei = 8; // pei - permanent equipment ID. Variable length of 16 bytes. TS-23.003 - 6.4 Like IMEI or IMEISV
+  string gpsi = 9; //Lenght is 10 bytes + 2 bytes country code generic public subscription id.
+  string user_location_info = 10; //up to 6 byte. TS-23.003 - 28.15.3
+  string pcf_id = 11; //length 16 bytes as UUID "Universally Unique IDentifier" TS-29.571 to ietf rfc 4122
+  //Variable length of 20 bytes. TS-23.003 - 28.8 & TS-23.501 5.9.8
+  repeated LadnServiceArea ladn_service_area = 12; //Length is 10 bytes, like PLMN
+  bool pdu_session_status_notification_sub = 13;   //Subscription For PDU Session Status Notification true or false
+  DnnSelectionMode dnn_selection_mode = 14;       // TS-29-502 - 6.1.6.3.10
+  string trace_requirment = 15;                   // TS-29.571 5.6.
+  bytes integrity_protection_max_data_rate = 16;  //Length is 3 bytes, TS-24.501 9.11.4.7
+  PduSessionType pdu_session_type = 17;
+  SscMode ssc_mode = 18;
+  M5GSMCapability m5g_sm_capability = 19;  // max size go up to 15 bytes. TLV and 9.11.4.1.
+  uint32 max_pkt_filter_supported = 20;    //Maximum no. of supported packet filters
+  bool pdu_session_req_always_on = 21;     //Always-on PDU session requested TS-24.501 9.11.4.4
+  string sm_pdu_dn_req = 22;               //SM PDU DN request container. Max size up to 255. TLV and 9.11.4.16.
+  M5GSMCause m5gsm_cause = 23;
+  QosRules requested_qos_rules = 24; //TS-24501 - 9.11.4.13 minimum 7 and max 65538, variable length.
+}
+
+message SetSMSessionContext {
+  CommonSessionContext common_context = 1;  //Common message for 4g, 5g and WiFi.
+  RatSpecificContext   rat_specific_context = 2;
+}
+message SmContextVoid {
+
+  string output = 1;
```

Comment:
What is this output value here?

Reply:
This is temporary value to adapt gRPC communication, to avoid any crash. it would be removed once integration of AMF and SMF is done.

Reply:
Ok, please put that it will be removed after a while in a comment so I can remember this in the future :)

Reply:
Done.